### ISSUE TITLE
Cannot run on Google Colab

### ISSUE THREAD

**Issue by pratikwatwani** [modin-project/modin#881]
### Describe the problem

Trying to work on a dataset that has 40+ features and more than 600,000+ data points, and wanted to work with modin for an efficient system to load data set, however, I work with my team on a shared Google Colab Jupyter Notebook, and am unable to install the same.  
I tried simple pip install modin, but that didn't work, so I tried, pip install modin[dask], that as well threw an error!

### Source code / logs

ModuleNotFoundError                       Traceback (most recent call last)
/usr/local/lib/python3.6/dist-packages/modin/__init__.py in get_execution_engine()
27             try:
28                 import ray
29 

ModuleNotFoundError: No module named 'ray'

During handling of the above exception, another exception occurred:

ImportError                               Traceback (most recent call last)
2 frames
/usr/local/lib/python3.6/dist-packages/modin/__init__.py in get_execution_engine()
45                     ):
46                         raise ImportError(
47                             "Please `pip install modin[dask] to install compatible Dask version."
48                         )
49             else:

ImportError: Please `pip install modin[dask] to install compatible Dask version. 

### Installation Error

ERROR: google-colab 1.0.0 has requirement tornado~=4.5.0, but you'll have tornado 6.0.3 which is incompatible.
Installing collected packages: tornado, dask, distributed, modin
  Found existing installation: tornado 4.5.3
    Uninstalling tornado-4.5.3:
      Successfully uninstalled tornado-4.5.3
  Found existing installation: dask 1.1.5
    Uninstalling dask-1.1.5:
      Successfully uninstalled dask-1.1.5
  Found existing installation: distributed 1.25.3
    Uninstalling distributed-1.25.3:
      Successfully uninstalled distributed-1.25.3
Successfully installed dask-2.8.0 distributed-2.8.0 modin-0.6.3 tornado-6.0.3
WARNING: Upgrading ipython, ipykernel, tornado, prompt-toolkit or pyzmq can
cause your runtime to repeatedly crash or behave in unexpected ways and is not
recommended. If your runtime won't connect or execute code, you can reset it
with "Reset all runtimes..." from the "Runtime" menu.
WARNING: tornado > 4.5.0 is incompatible with ipykernel < 5.0
WARNING: The following packages were previously imported in this runtime:
  [tornado]
Run "pip install -U ipykernel" before restarting to avoid repeated crashes.

**Comment by manesioz** [comment]
After restarting my runtime, I was able to get it to work by running: 

```python
!pip install -U ipykernel
!pip install modin[dask]
```

Can you try this?

**Comment by pratikwatwani** [comment]
> After restarting my runtime, I was able to get it to work by running:
> 
> ```
> !pip install -U ipykernel
> !pip install modin[dask]
> ```
> 
> Can you try this?

Hi, I did try this, but unfortunately I cannot seem to install modin on  Colab!

**Comment by devin-petersohn** [comment]
@pratikwatwani What was the error message you saw?

**Comment by pratikwatwani** [comment]
> @pratikwatwani What was the error message you saw?

It's the same thing as I posted initially. I tried 
`!pip install -U ipykernel
  !pip install modin[dask]`

The above commands install median on colab with some warnings, but when I try to import modin, I keep getting message for installing it via dask, along with "no module named ray"

**Comment by pratikwatwani** [comment]
@devin-petersohn, @manesioz

I got it working with the solution @manesioz provided, however, it required me to make a multiple restart of my runtime, which is absolutely weird.

When I installed it initially, and restarted the runtime it still did not recognize the package, however after repeating the process again, I did a multiple restart which seemed to solve my problem. 

I don't know why I had to that though.

**Comment by pratikwatwani** [comment]
@devin-petersohn, @manesioz

I got it working with the solution @manesioz provided, however, it required me to make a multiple restart of my runtime, which is absolutely weird.

When I installed it initially, and restarted the runtime it still did not recognize the package, however after repeating the process again, I did a multiple restart which seemed to solve my problem. 

I don't know why I had to that though.

**Comment by pratikwatwani** [comment]
@devin-petersohn, @manesioz

I got it working with the solution @manesioz provided, however, it required me to make a multiple restart of my runtime, which is absolutely weird.

When I installed it initially, and restarted the runtime it still did not recognize the package, however after repeating the process again, I did a multiple restart which seemed to solve my problem. 

I don't know why I had to that though.
